tests/Stormpath/Tests/Resource/AccountCreationPolicyTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Stormpath\Tests\Resource;
+
+
+use Stormpath\Stormpath;
+
+class AccountCreationPolicyTest extends \Stormpath\Tests\BaseTest {
+
+    private static $directory;
+
+    private static $acp;
+
+    public static function setUpBeforeClass()
+    {
+        parent::setUpBeforeClass();
+        self::$directory = \Stormpath\Resource\Directory::create(array('name' => 'Another random directory' .md5(time().microtime().uniqid())));
+        self::$acp = self::$directory->accountCreationPolicy;
+    }
+
+    /**
+     * @test
+     */
+    public function it_should_allow_changing_verification_email_status()
+    {
+        $this->assertEquals(STORMPATH::DISABLED, self::$acp->verificationEmailStatus);
+
+        self::$acp->verificationEmailStatus = STORMPATH::ENABLED;
+
+        $this->assertEquals(STORMPATH::ENABLED, self::$acp->verificationEmailStatus);
+
+        self::$acp->verificationEmailStatus = STORMPATH::DISABLED;
+
+        $this->assertEquals(STORMPATH::DISABLED, self::$acp->verificationEmailStatus);
+    }
+
+    /**
+     * @test
+     */
+    public function it_should_allow_changing_verification_success_email_status()
+    {
+        $this->assertEquals(STORMPATH::DISABLED, self::$acp->verificationSuccessEmailStatus);
+
+        self::$acp->verificationSuccessEmailStatus = STORMPATH::ENABLED;
+
+        $this->assertEquals(STORMPATH::ENABLED, self::$acp->verificationSuccessEmailStatus);
+
+        self::$acp->verificationSuccessEmailStatus = STORMPATH::DISABLED;
+
+        $this->assertEquals(STORMPATH::DISABLED, self::$acp->verificationSuccessEmailStatus);
+    }
+
+    /**
+     * @test
+     */
+    public function it_should_allow_changing_welcome_email_status()
```

Comment:
All new test classes I made have this format.  If I add a test to an existing class, it follows old conventions.  I will go back through and re-name all tests to the new convention.